src/main/java/com/coursemngsys/coursemanagementsystem/temp/User.java

```diff
@@ -0,0 +1,58 @@
+package com.coursemngsys.coursemanagementsystem.temp;
+
+import java.util.ArrayList;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+/**
+ *
+ * @author 56130
+ */
+public class User {
+
+    private int id ;
+    private String firstName;
+    private String lastName;
+    private int age;
+    private String profession;
+    private ArrayList<User> children;
+
+    public User(int id, String firstName, String lastName, int age) {
```

Comment:
Konstruktoriaus parametruose trūksta "profession" atributo.
```suggestion
    public User(int id, String firstName, String lastName, int age, String profession) {
```